service/service.py

```diff
@@ -127,6 +127,14 @@ def list_suppliers():
     results = [supplier.serialize() for supplier in suppliers]
     return make_response(jsonify(results), status.HTTP_200_OK)
 
+######################################################################
+# UPDATE AN EXISTING SUPPLIER
+######################################################################
+@app.route('/suppliers/<int:supplier_id>', methods=['PUT'])
+def update_suppliers(supplier_id):
+    return str(supplier_id)
```

Comment:
I guess it is supposed to return a jsonify result of this updated supplier?

Reply:
Yes, it will. currently this submission just creates a route without real return.